### DISCUSSION TITLE
2011/07/07/lucene-contrib%EF%BC%88%E8%B4%A1%E7%8C%AE%E8%BD%AF%E4%BB%B6%EF%BC%89%E4%B8%8B%E5%B8%B8%E7%94%A8%E6%A8%A1%E5%9D%97%E7%9A%84%E5%8A%9F%E8%83%BD

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#337] [Announcements]
# 2011/07/07/lucene-contrib%EF%BC%88%E8%B4%A1%E7%8C%AE%E8%BD%AF%E4%BB%B6%EF%BC%89%E4%B8%8B%E5%B8%B8%E7%94%A8%E6%A8%A1%E5%9D%97%E7%9A%84%E5%8A%9F%E8%83%BD

Lucene 是一个开源的搜索引擎，它的基本功能是对文本建立索引，帮助快速找到包含关键字的文档。Lucene被广泛的应用到了各种软件中，目前当需要在软件中添加搜索功能时，工程师都首先会想到借助Lucene。虽然Lucene只具备基本的文本索引和搜索功能，但广大工程师在实际应用中对它进行了扩展，添加了大量高级功能，并且将他们的改进贡献了出来。一些最为常用的功能也被收录到了Lucene的发布包中，被放置在contrib文件夹下。

https://blog.qizhen.xyz/2011/07/07/lucene-contrib%EF%BC%88%E8%B4%A1%E7%8C%AE%E8%BD%AF%E4%BB%B6%EF%BC%89%E4%B8%8B%E5%B8%B8%E7%94%A8%E6%A8%A1%E5%9D%97%E7%9A%84%E5%8A%9F%E8%83%BD

**Comment by ruanqizhen** [comment]
之前博客下的留言：

怎么获取前面板句柄？
2011年07月8日 16:34 编辑
阮老师，怎么显示属性FP.OSWindow?

回复
ruanqizhen
2011年07月11日 10:57 编辑
这个属性不是每个版本的LabVIEW都有的。如果你在别人的程序里看到它了，直接拿来用就可以了。否则就没办法了。

回复
